build.cake

```diff
@@ -1,3 +1,6 @@
+#tool "KuduSync.NET"
+#addin "Cake.Kudu"
```

Comment:
Recommendation here would be to pin addin version, example:
```
#addin nuget:?package=Cake.Foo&version=1.2.3
```
https://cakebuild.net/docs/fundamentals/preprocessor-directives